app/lib/user_attributes.rb

```diff
@@ -1,5 +1,5 @@
 class UserAttributes
-  CONFIG_KEYS = %w[is_stored_locally is_cached_locally permissions].freeze
+  CONFIG_KEYS = %w[type permissions].freeze
```

Comment:
Alternatively we could use an Enum here?

```
  enum :type, [ :local, :remote, :cached ]
```

That seems to let ActiveRecord give us a bunch of cool methods like `.remote?` or `.local?` or `.cached?`